### DISCUSSION TITLE
Use of fvm vs. fvmb

### DISCUSSION THREAD

**Discussion by jPlasma** [fppimenta/rheoTool#89] [General]
Hello! I'm new to rheoTool and am using it to write OF based solvers with couplings terms. Thank you for the great library!

I have following questions:
1) For coupling terms, can insertEquation function be supplied fvm operators from OF or should only the fvmb operators be used?
2) If fvm operators from OF can be used, when/why are the custom operators defined via fvmb required?

For reference:
Example use case of fvm in insertEquation for coupling terms? I noticed [NernstPlanckCoupled.C](https://github.com/fppimenta/rheoTool/blob/3ee9995f6635429c31f6a99beed8334941d57354/of90/src/libs/EDFModels/models/NernstPlanck/NernstPlanckCoupled/NernstPlanckCoupled.C#L471-L478) uses fvm operator:
```
   // ci_psi
   cps.insertEquation
   (
     ci.name(),
     psi_.name(),
     fvm::laplacian(-ci*cf, psiCorr, "laplacian(elecM)")	 	 	
   );   
 }
```

Example of fvmb for coupling terms:
```
// div(U)
cps->insertEquation
(
 p.name(),
 U.name(),
 fvmb::div(-1., U)
);
```

Thank you!!

**Comment by fppimenta** [comment]
It's not an issue but a question.

fvmb is used (created) when the operator is not available in fvm.

**Reply by jPlasma** [reply]
Thank you!!
